docs/sdk/src/guides/enable_elastic_scaling_mvp.rs

```diff
@@ -108,8 +147,10 @@
 //! produce per relay chain block (in direct correlation with the number of acquired cores). This
 //! should be either 1 (no scaling), 2 or 3. This is called the parachain velocity.
 //!
-//! If you configure a velocity which is different from the number of assigned cores, the measured
-//! velocity in practice will be the minimum of these two.
+//! <div class="warning">If you configure a velocity which is different from the number of assigned
+//! cores, the measured velocity in practice will be the minimum of these two. However, be mindful
+//! that if the velocity is higher than the number of assigned cores, it's possible that
+//! <a href="https://github.com/paritytech/polkadot-sdk/issues/6667"> only a subset of the collator set will be authoring blocks.</a></div>
```

Comment:
The question is why do we need to configure a velocity at all, seems redundant.

Reply:
Once the slot based collator can produce multiple blocks per slot we should also add that we recommend slot durations of at least 6s, preferably even 12. (better censorship resistance)